src/bin/edit/localization.rs

```diff
@@ -464,6 +489,7 @@ const S_LANG_LUT: [[&str; LangId::Count as usize]; LocId::Count as usize] = [
         /* ko      */ "상태 표시줄로 포커스 이동",
         /* pt_br   */ "Focar barra de status",
         /* ru      */ "Фокус на строку состояния",
+        /* tr      */ "Durum çubuğuna odaklan",
```

Comment:
For consistency and aesthetics, it might be a better option to stick to Title Case, “Durum Çubuğuna Odaklan”.